netweaver/install_db.sls

```diff
@@ -49,6 +56,7 @@ netweaver_install_{{ instance_name }}:
     - additional_dvds: {{ netweaver.additional_dvds }}
     - require:
       - create_db_inifile_{{ instance_name }}
+      - check_sapprofile_directory_exists_{{ instance_name }}
```

Comment:
Maybe we should put this `require` condition in the previous `wait_for_hana` state. To avoid running this super long check if previous states failed already.

Reply:
I think `wait_for_hana` and `check_sapprofile_directory_exists` are not dependent, logically as well. But creating the dependency will not harm, and may help reduce the long check as you mentioned. SO i will update the `require` condition for `wait_for_hana`